.github/workflows/test.yml

```diff
@@ -0,0 +1,45 @@
+name: Run tests
+on: pull_request
+jobs:
+  test-polyfill:
+    strategy:
+      matrix:
+        node: [10.x, 12.x, 14.x, 15.x]
+        os: [ubuntu-latest, windows-latest, macos-latest]
```

Comment:
Is this running everything 12 times? That sounds like overkill.

Reply:
What about dropping 10.x?